src/components/vm/nics/nicAdd.tsx

```diff
@@ -200,6 +215,12 @@ export class AddNIC extends React.Component<AddNICProps, AddNICState> {
             return;
         }
 
+        const validation = validateDialogBodyValues(this.state);
+        if (validation) {
+            this.setState({ validation, doOnlineValidation: true });
+            return;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-66f0c2f5-20251104-070631-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)